jax/_src/numpy/ufuncs.py

```diff
@@ -365,9 +365,38 @@ def rint(x: ArrayLike, /) -> Array:
   return lax.round(x, lax.RoundingMethod.TO_NEAREST_EVEN)
 
 
-@implements(np.copysign, module='numpy')
 @jit
 def copysign(x1: ArrayLike, x2: ArrayLike, /) -> Array:
+  """Copies the sign of each element in ``x2`` to the corresponding element in ``x1``.
+
+  LAX-backend implementation of :func:`numpy.copysign`.
+
+  Args:
+    x1: Input array
+    x2: The array who's elements will be used to determine the sign
+
+  Returns:
+    An array-like object containing the potentially changed elements of ``x1``, this
+    function is broadcastable.
```

Comment:
Returns an array, not an array-like

Mention that elements will be promoted to inexact dtype

Rather than saying "this function is broadcasteable", maybe say "array of shape `jnp.broadcast_shapes(x1.shape, x2.shape)`" or something similar